ada/language/ast.py

```diff
@@ -10306,6 +10306,9 @@ def xref_equation():
             # Lal checkers specific
             rel_name == 'Model', Entity.model_attr_equation,
 
+            rel_name == 'Callable',
+            Entity.prefix.sub_equation & bool_bind(Self.type_var),
+
```

Comment:
Is the testcase missing, or is it somewhere else?

Reply:
Good catch, it was forgotten!